org.lflang/src/org/lflang/DefaultErrorReporter.java

```diff
@@ -0,0 +1,62 @@
+package org.lflang;
+
+import org.eclipse.emf.ecore.EObject;
+
+import java.nio.file.Path;
+
+/**
+ * Simple implementation of the ErrorReport interface that simply prints to
+ * standard out.
+ */
+public class DefaultErrorReporter implements ErrorReporter {
+
+    private boolean errorsOccurred = false;
+
```

Comment:
The static member was removed. I'm sure you didn't do this by accident, but could you explain why?

Reply:
Yes it's because of the field we can see here